### PR TITLE
Refactor experiment config

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Refactor experiments into an object with `id` as key
- Better logging of assigned experiment

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/services/Tracking/experiment.helpers.ts [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
-  EXPERIMENT_COOKIE_NAME,
-  Experiments,
-  type Experiment,
-  type ExperimentVariant,
+  CurrentExperiments,
+  Experiment,
+  ExperimentConfig,
+  ExperimentVariant,
 } from './experiment.constants'
+import { CURRENT_EXPERIMENTS, EXPERIMENT_COOKIE_NAME, Experiments } from './experiment.constants'
```
Comment (guilhermespopolin) [inline apps/store/src/services/Tracking/experiment.helpers.ts:8]: Should we use a single import statement for these?

**Review by guilhermespopolin — APPROVED** [review]
Nice additions :)

File: apps/store/src/services/Tracking/experiment.helpers.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/services/Tracking/experiment.helpers.ts:8]: Ah, that's my eslint not working properly 😔

Comment (gustaveen) [reply apps/store/src/services/Tracking/experiment.helpers.ts:8]: Might be my eslint not working properly. But shouldn't the new rule split up types and imports?

Comment (guilhermespopolin) [reply apps/store/src/services/Tracking/experiment.helpers.ts:8]: I don't know for sure now 😬 I mean, I remember the new rule being keep everything on a single import statement

Comment (gustaveen) [reply apps/store/src/services/Tracking/experiment.helpers.ts:8]: Got it working now, so lint should fix it 👍

**Pull request merged**